### PR TITLE
Add image and README for Prow config updater tool

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
1. Add image for Prow config updater tool
2. Add README
3. A couple of typo bug fixes

/cc @chaodaiG

### CODE REVIEW

File: tools/prow-config-updater/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- `--github-token-file` [Required] specifies the path of file containing Github
+  token for most of important interactions with Github.
+- `--git-userid` [Required] specifies the Github ID of user for hosting fork,
+  i.e. Github ID of bot.
+- `--git-username` [Optional] specifies the username to use on the git commit.
+  Requires --git-email.
+- `--git-email` [Optional] specifies the email to use on the git commit.
+  Requires --git-username.
+- `--comment-github-token-file` [Required] specifies the path of file containing Github
+  token for adding comments on the pull requests.
```
Comment (mattmoor) [inline tools/prow-config-updater/README.md]: Format markdown:
```suggestion
- `--comment-github-token-file` [Required] specifies the path of file containing
  Github token for adding comments on the pull requests.
```

**Comment by chaodaiG** [comment]
/lgtm

/hold
Left a nit, feel feel do it in another PR if want to get this tested sooner

**Comment by chizhg** [comment]
> /lgtm
> 
> /hold
> Left a nit, feel feel do it in another PR if want to get this tested sooner

Hmm, did you forget to add the nit? I cannot find it..

**Comment by chaodaiG** [comment]
> > /lgtm
> > /hold
> > Left a nit, feel feel do it in another PR if want to get this tested sooner
> 
> Hmm, did you forget to add the nit? I cannot find it..

forgot :)

File: tools/prow-config-updater/pullrequest.go [hunk 2]
```diff
@@ -51,9 +52,9 @@ func (gc *GitHubMainHandler) getLatestPullRequest() (*github.PullRequest, error)
 		return nil, fmt.Errorf("error getting the last commit ID: %v", err)
 	}
 	// As we always use squash in merging PRs, we can get the pull request with the commit ID.
-	pr, err := gc.client.GetPullRequestByCommitID(config.OrgName, config.RepoName, ci)
+	pr, err := gc.client.GetPullRequestByCommitID(config.OrgName, config.RepoName, strings.Trim(ci, "\n"))
```
[the remark this answers is not in the document]
Comment (chizhg) [reply tools/prow-config-updater/pullrequest.go]: Done.

**Pull request merged**